subscription/tests.py

```diff
@@ -406,6 +406,14 @@ def test_send_3_part_message_1_en_subscription(self):
         self.assertEquals(subscriber_updated.next_sequence_number, 2)
         self.assertEquals(subscriber_updated.process_status, 0)
 
+    def test_noop_message_3_en_subscription(self):
+        subscription = Subscription.objects.get(pk=7)
+        result = processes_message.delay(subscription.id, self.sender)
+        self.assertEqual(result.get(), [])
+        subscriber_updated = Subscription.objects.get(pk=7)
+        self.assertEquals(subscriber_updated.next_sequence_number, 3)
+        self.assertEquals(subscriber_updated.process_status, 2)
```

Comment:
Please add a comment here to indicate why the next_sequence_number is still `3` - i.e. the messageset is finished, and add an assert before line 411 to indicate what the `next_sequence_number` and `process_status` is before the task is run.